pom.xml

```diff
@@ -47,6 +47,12 @@
     </dependency>
 
     <dependency>
+      <groupId>org.slf4j</groupId>
```

Comment:
I think this is fine as it's just an API and is not bringing in any heavyweight loggers. SLF4J have their own opinion on this: http://slf4j.org/faq.html#optional_dependency